[docs] Adds preview and production update docs (#30667)
expo/expo: docs/pages/deploy/send-over-the-air-updates.mdx

---
title: Send over-the-air updates
description: Learn how to send over-the-air updates to push critical bug fixes and improvements to your users.
---

import { Terminal } from '~/ui/components/Snippet';

> You can send over-the-air updates containing critical bug fixes and improvements to your users.

Reviewer: ```suggestion
You can send over-the-air (OTA) updates containing critical bug fixes and improvements to your users.
```

Should we add the abbreviate form here?

Author: I did not since there are no more instances below this of "over-the-air", so there's nowhere to use it. So, I think no unless we add more instances of "over-the-air" later on in the doc. Then I think OTA would be okay to add.

## Get started

> > If you've published [previews](/review/share-previews-with-your-team/) or created a [build](/deploy/build-project/) before, you may have already set up updates.

Reviewer: ```suggestion
> If you've published [previews](/review/share-previews-with-your-team/) or created a [build](/deploy/build-project/) before, you may have already set up updates. Skip this section.
```

To set up updates, run the following [EAS CLI](/develop/tools/#eas-cli) command:

<Terminal cmd={['$ eas update:configure']} />

> Once the command completes, you'll need to make new builds.

Reviewer: ```suggestion
After the command completes, you'll need to make new builds before moving to the next section.
```

Let's add the context about creating the build before taking the next step here.

## Send an update

To send an update, run the following [EAS CLI](/develop/tools/#eas-cli) command:

<Terminal cmd={['$ eas update --channel production']} />

> This command will create an update and make it available to builds of your app that are configured to receive updates on the `production` channel, which is defined in [**eas.json**](/eas/json/#channel).

Reviewer: ```suggestion
This command will create an update and make it available to builds of your app that are configured to receive updates on the `production` channel. This channel is defined in [**eas.json**](/eas/json/#channel).
```

Let's split the last part to improve readability.
